recipes/gnu-getopt/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "gnu-getopt" %}
+{% set version = "1.1.6" %}
+{% set md5 = "4eaff9e6d8b2504a2e0d29ff0e9fd956" %}
+
+package: 
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz  # [osx]
+  md5: {{ md5 }}  # [osx]
+  url: http://frodo.looijaard.name/system/files/software/getopt/getopt-{{ version }}.tar.gz  # [osx]
+  patches:
+    - Makefile.osxpatch  # [osx]
+
+build:
+  number: 0
+  skip: True  # [not osx]
+
+requirements:
+  build:
+    - gcc  # [osx]
+    - gettext  # [osx]
+  run:
+    - libgcc  # [osx]
+    - gettext  # [osx]
+
+test:
+    commands:
+        - '$PREFIX/bin/gnu-getopt -V'  # > /dev/null'
+
+about:
+    home: 'http://software.frodo.looijaard.name/getopt/'
+    license: "GPL-2.0"
```

Comment:
No need for quotes with either of these.

Reply:
The quotes do not make a difference here, and honestly this is sort of pickiness that has prevented me from adding packages to conda-forge, despite being a core contributor of many packages to bioconda. The more substantive changes in this PR I can address, but two bytes for quotes are really a blocking issue?